src/ngraph/op/fused/depth_to_space.cpp

```diff
@@ -83,5 +107,18 @@ shared_ptr<Node> op::DepthToSpace::copy_with_new_args(const NodeVector& new_args
     {
         throw ngraph_error("Incorrect number of new arguments");
     }
-    return make_shared<DepthToSpace>(new_args.at(0), m_blocksize);
+    return make_shared<DepthToSpace>(new_args.at(0), m_mode, m_blocksize);
+}
+
+op::DepthToSpace::DepthToSpaceMode
+    op::DepthToSpace::depth_to_space_mode_from_string(const std::string& sort) const
```

Comment:
Non blocking:
Name of the operator is already in a class name.
```suggestion
    op::DepthToSpace::mode_from_string(const std::string& sort) const
```